arm/ui/comments.json

```diff
@@ -48,13 +48,15 @@
   "SET_MEDIA_OWNER": "# Don't bother. This does not do anything yet",
   "CHOWN_USER": "",
   "CHOWN_GROUP": "",
-  "RIPMETHOD": "# Method of MakeMKV to use for Blu Ray discs.  Options are \"mkv\" or \"backup\".\n# backup decrypts the dvd and then copies it to the hard drive.  This allows HandBrake to apply some of it's\n# analytical abilities such as the main-feature identification.  This method seems to offer success on bluray \n# discs that fail in \"mkv\" mode. *** NOTE: MakeMKV only supports the backup method on BluRay discs.  ARM does\n# not use MakeMKV for DVDs.   ",
+  "RIPMETHOD": "# Method of MakeMKV to use for Blu Ray discs.  Options are \"mkv\", \"backup\" or \"backup_dvd\".\n# backup decrypts the dvd and then copies it to the hard drive.  This allows HandBrake to apply some of it's\n# analytical abilities such as the main-feature identification.  This method seems to offer success on bluray \n# discs that fail in \"mkv\" mode. *** NOTE: MakeMKV only supports the backup or backup_dvd method on BluRay discs.\n# backup_dvd forces arm to extract the dvd with MakeMKV prior to the Handbrak
+e step",
```

Comment:
This newline breaks the json file, which in turn breaks the ability to save ripper settings.

Reply:
fixed